test/evergreen.yml

```diff
@@ -4329,6 +4770,12 @@ buildvariants:
     - name: ".stress-test-sanitizer-3"
     - name: ".stress-test-sanitizer-4"
     - name: format-abort-recovery-stress-test
+    - name: ".stress-test-1-nonstandalone"
+    - name: ".stress-test-2-nonstandalone"
+    - name: ".stress-test-3-nonstandalone"
+    - name: ".stress-test-4-nonstandalone"
+    - name: ".stress-test-no-barrier-nonstandalone"
+    - name: format-abort-recovery-stress-test-nonstandalone
```

Comment:
It feels a bit overkill to have this many non-standalone stress tests running on the ASAN builder. Can we reduce it to 1 or 2?